src/components/layout/Undernames/Undernames.tsx

```diff
@@ -250,11 +251,11 @@ function Undernames() {
         <div className="modal-container">
           <DialogModal
             title={
-              action === 'create'
+              action === UNDERNAME_TABLE_ACTIONS.CREATE
                 ? 'Create Undername'
-                : action === 'edit'
+                : action === UNDERNAME_TABLE_ACTIONS.EDIT
                 ? `Edit ${selectedRow?.name}`
-                : action === 'remove'
+                : action === UNDERNAME_TABLE_ACTIONS.REMOVE
                 ? `Remove ${selectedRow?.name}`
                 : ''
```

Comment:
we've found burying the logic in a component is sometimes hard to debug. consider moving to a helper function if not going to use an object

Reply:
added